tests/oclo/Makefile

```diff
@@ -0,0 +1,14 @@
+.PATH:	${SRCTOP}/cddl/contrib/opensolaris/tests/oclo
+
+TESTSDIR=	${TESTSBASE}/cddl/oclo
+
+PLAIN_TESTS_C=	oclo oclo_errors ocloexec_verify
+
+_ALLOW_ABSOLUTE_OBJ_PATH= 1
+
+SRCS.oclo=	oclo.c \
+		${SRCTOP}/crypto/openssh/openbsd-compat/recallocarray.c
```

Comment:
We have lib/libopenbsd, could you please add it there instead and link it?

Reply:
I couldn't fix this one.  This library lacks `recallocarray`:

`ld: error: undefined symbol: recallocarray`

Reply:
> Right, I'm suggesting that you add the function to that library as well.

Fixed. 

Eventually these must be adapted to use libopenbsd as well:
- usr.bin/mandoc/Makefile
- lib/libfido2/Makefile
- lib/libdiff/Makefile

Reply:
reallocarray was added to POSIX so we should probably add it to libc. https://pubs.opengroup.org/onlinepubs/9799919799.2024edition/functions/reallocarray.html

Reply:
> reallocarray was added to POSIX so we should probably add it to libc. https://pubs.opengroup.org/onlinepubs/9799919799.2024edition/functions/reallocarray.html

We already have reallocarray.  This one is recallocarray :)